src/components/Kv/KvCarousel.vue

```diff
@@ -29,53 +166,288 @@ export default {
+				direction = 'right';
+			} else if (this.currentIndex === this.slides.length - 1 && index === 0) { // last slide to first
+				direction = 'left';
+			} else if (index < this.currentIndex) {
+				direction = 'right';
+			} else if (index > this.currentIndex) {
+				direction = 'left';
 			}
-			return 0;
+			this.transitionName = `kv-slide-${direction}`;
+
+			// wait for the transition to be applied
+			this.$nextTick(() => {
+				this.intervalTimerCurrentTime = 0;
+				this.slides[this.currentIndex].hide();
+				this.currentIndex = index;
+				this.slides[this.currentIndex].show();
+
+				/**
+				 * The index of the slide that the carousel has changed to
+				 * @event change
+				 * @type {Event}
+				 */
+				this.$emit('change', this.currentIndex);
+			});
 		},
-		getPreviousIndex() {
-			const previousSlideIndex = this.currentIndex - 1;
-			if (previousSlideIndex >= 0) {
-				return previousSlideIndex;
-			}
-			return this.slides.length - 1;
-		}
 	}
 };
 </script>
 
 <style lang="scss" scoped>
-@import 'settings';
+@import "settings";
+
+$arrow-width: rem-calc(41);
+$arrow-margin: rem-calc(8);
+$circle-indicator-height: rem-calc(16);
+$circle-indicator-margin: rem-calc(8);
+$bar-indicator-height: rem-calc(4);
+$bar-indicator-margin: rem-calc(4);
 
 .kv-carousel {
 	position: relative;
+	display: flex;
 	overflow: hidden;
 	width: 100%;
-	height: 100%;
+	// you'll aways need to add either a height: 1234px; or height: 0; padding-bottom: 25% when you put it on the page.
+
+	&__slides {
+		position: absolute;
+		top: 0;
+		right: #{$arrow-width + $arrow-margin};
+		bottom: #{$circle-indicator-height + $circle-indicator-margin * 2};
+		left: #{$arrow-width + $arrow-margin};
+		flex: 1;
+		background: green; // TODO: colors
+		overflow: hidden;
+	}
+
+	&__arrows {
+		position: absolute;
+		top: 0;
+		left: 0;
+		bottom: #{$circle-indicator-height + $circle-indicator-margin * 2};
+		right: 0;
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		pointer-events: none;
+	}
+
+	&__arrows-btn {
+		background: $kiva-text-light;
+		fill: #fff;
+		border-radius: 50%;
+		width: $arrow-width;
+		height: $arrow-width;
+		padding: 0.75rem;
+		pointer-events: auto;
+
+		&--right {
+			transform: rotate(270deg);
+		}
+
+		&--left {
+			transform: rotate(90deg);
+		}
+
+		&:hover,
+		&:focus {
+			background: $dark-charcoal;
+		}
+	}
+
+	&__arrows-icon {
+		fill: #fff;
+	}
+
+	&__indicator {
+		position: absolute;
+		left: #{$arrow-width + $arrow-margin};
+		right: #{$arrow-width + $arrow-margin};
+		bottom: 0;
+		display: flex;
+		justify-content: center;
+		pointer-events: none;
+	}
+
+	&__indicator-btn {
+		margin: 0.5rem;
+		margin-left: 0;
+		width: 1rem;
+		height: 1rem;
+		border-radius: 50%;
+		background: $kiva-text-dark;
+		pointer-events: auto;
+		position: relative;
+		overflow: hidden;
+
+		&:hover,
+		&:focus {
+			background: yellow; // TODO: colors
```

Comment:
How do you propose we handle the colors?

Reply:
Good question, I'd like to get this merged (after squash) before the branch cut. Then today work with Aaron to get the hover/focus/progressbars etc colors sorted out.